src/Integration.UnitTests/LocalServices/UnboundProjectFinderTests.cs

```diff
@@ -130,9 +133,21 @@ public UnboundProjectFinder CreateTestSubject()
                 sp.RegisterService(typeof(IProjectSystemHelper), projectSystemHelper.Object);
                 sp.RegisterService(typeof(IConfigurationProviderService), configProviderMock.Object);
 
-                var testSubject = new UnboundProjectFinder(sp, new TestLogger(), projectBinderFactoryMock.Object);
+                var testSubject = new UnboundProjectFinder(sp, new TestLogger(), projectBinderFactoryMock.Object, CreateThreadHandlingWithRunPassthrough().Object);
                 return testSubject;
             }
+
+            private static Mock<IThreadHandling> CreateThreadHandlingWithRunPassthrough()
```

Comment:
FYI I added some more methods to the interface in a later commit, at which point I create a helper class in the `TestInfrastructure` project (simpler than mocking).

I can't remember if I removed this use of a mocked version, though.

Reply:
I'm looking at the latest commit and it's still here, so I think it can be replaced.